lib/MusicBrainz/Server/Controller/CDTOC.pm

```diff
@@ -413,10 +413,32 @@ sub move : Local Edit
         $c->stash( template => 'cdtoc/move_search.tt' );
 
         if ($c->form_submitted_and_valid($search_release, $c->req->query_params)) {
+            my $query = $search_release->field('query')->value;
+            my ($mbid) = $query =~ m/(
+                [\da-f]{8} -
+                [\da-f]{4} -
+                [\da-f]{4} -
+                [\da-f]{4} -
+                [\da-f]{12}
+            )/ax;
+
             my $releases = $self->_load_paged($c, sub {
-                $c->model('Search')->search('release', $search_release->field('query')->value, shift, shift,
+                if (defined $mbid) {
+                    $c->stash->{was_mbid_search} = 1;
```

Comment:
We should handle `was_mbid_search` the same way we for the attach pages, I guess?

Reply:
This is sent to an attach page, so I think that it requires no further changes. Correct me if wrong :)